app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -0,0 +1,86 @@
+require 'barby'
+require 'barby/barcode/code_128'
+require 'barby/outputter/png_outputter'
+
+module Idv
+  module InPerson
+    class ReadyToVerifyPresenter
+      def initialize(enrollment:)
+        @enrollment = enrollment
+      end
+
+      def barcode_data_url
+        "data:image/png;base64,#{Base64.strict_encode64(barcode_image_data)}"
+      end
```

Comment:
As I'm beginning to work on LG-6343 (email notification) which is also meant to include an image of the barcode, I may plan to revise the output, since data URLs are not well-supported in email clients ([[1]](https://stackoverflow.com/questions/6070196/what-is-data-uri-support-like-in-major-email-client-software)[[2]](https://stackoverflow.com/questions/3279523/base64-images-to-gmail/12786336#comment58993716_12786336)[[3]](https://stackoverflow.com/questions/46783599/base64-encoded-image-is-not-showing-in-gmail)). The Barby gem supports a number of "[Outputters](https://github.com/toretore/barby/wiki/Outputters)", and the [HTML outputter](https://github.com/toretore/barby/wiki/Outputters#html) seems to be the most promising for email compatibility, and wouldn't be a big lift to swap in to use in the web layout for consistency.

I probably won't change it here now, but will revisit with the pull request for LG-6343.

Reply:
(A preview of this rework can be seen at #6585)